### PR TITLE
docker-ps: add Indonesian translation

### PR DESCRIPTION
**Pull request by fadharpra**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by fadharpra** [comment]
Hi @kbdharun and @reinhart1010, kindly review my request

File: pages.id/common/docker-ps.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`docker ps --all --filter="exited={{kode}}"`
+
+- Memilah kontainer berdasarkan status (created, running, removing, paused, exited, dan dead):
+
+`docker ps --filter="status={{status}}"`
+
+- Memilah kontainer yang mengaitkan suatu volume tertentu atau memiliki volume yang terpasang pada jalur tertentu:
+
+`docker ps --filter="volume={{path/ke/direktori}}" --format "table {{.ID}}\t{{.Image}}\t{{.Names}}\t{{.Mounts}}"`
```
Comment (kbdharun) [inline pages.id/common/docker-ps.md:36]: Thanks for your contribution, I have a suggestion for this page.

```suggestion
`docker ps --filter="volume={{jalan/menuju/direktori}}" --format "table {{.ID}}\t{{.Image}}\t{{.Names}}\t{{.Mounts}}"`
```
Ref. [Common Arguments](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)

File: pages.id/common/docker-ps.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# docker ps
+
+> Menampilkan daftar kontainer Docker.
```
Comment (reinhart1010) [inline pages.id/common/docker-ps.md:3]:
```suggestion
> Tampilkan daftar kontainer Docker.
```
Imperative mood, see #8576

File: pages.id/common/docker-ps.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+> Informasi lebih lanjut: <https://docs.docker.com/engine/reference/commandline/ps/>.
+
+- Menampilkan kontainer Docker yang sedang berjalan saat ini:
```
Comment (reinhart1010) [inline pages.id/common/docker-ps.md:6]:
```suggestion
- Tampilkan kontainer Docker yang sedang berjalan saat ini:
```

File: pages.id/common/docker-ps.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`docker ps`
+
+- Menampilkan semua kontainer Docker (yang berjalan dan yang berhenti):
```
Comment (reinhart1010) [inline pages.id/common/docker-ps.md:10]:
```suggestion
- Tampilkan semua kontainer Docker (yang berjalan dan yang berhenti):
```

File: pages.id/common/docker-ps.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`docker ps --all`
+
+- Menampilkan kontainer yang dibuat terakhir (termasuk semua status):
```
Comment (reinhart1010) [inline pages.id/common/docker-ps.md:14]:
```suggestion
- Tampilkan kontainer yang dibuat terakhir (termasuk semua status):
```

File: pages.id/common/docker-ps.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`docker ps --latest`
+
+- Memilah kontainer yang mengandung substring dalam namanya:
```
Comment (reinhart1010) [inline pages.id/common/docker-ps.md:18]:
```suggestion
- Pilah kontainer yang mengandung substring dalam namanya:
```

File: pages.id/common/docker-ps.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+`docker ps --filter="name={{nama}}"`
+
+- Memilah kontainer yang memiliki gambar yang sama sebagai leluhur:
```
Comment (reinhart1010) [inline pages.id/common/docker-ps.md:22]:
```suggestion
- Pilah kontainer yang memiliki gambar yang sama sebagai leluhur:
```

File: pages.id/common/docker-ps.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+`docker ps --filter "ancestor={{image}}:{{tag}}"`
+
+- Memilah kontainer berdasarkan kode status keluar (exit status code):
```
Comment (reinhart1010) [inline pages.id/common/docker-ps.md:26]:
```suggestion
- Pilah kontainer berdasarkan kode status keluar (exit status code):
```

File: pages.id/common/docker-ps.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+`docker ps --all --filter="exited={{kode}}"`
+
+- Memilah kontainer berdasarkan status (created, running, removing, paused, exited, dan dead):
```
Comment (reinhart1010) [inline pages.id/common/docker-ps.md:30]:
```suggestion
- Pilah kontainer berdasarkan status (created, running, removing, paused, exited, dan dead):
```

File: pages.id/common/docker-ps.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+`docker ps --filter="status={{status}}"`
+
+- Memilah kontainer yang mengaitkan suatu volume tertentu atau memiliki volume yang terpasang pada jalur tertentu:
```
Comment (reinhart1010) [inline pages.id/common/docker-ps.md:34]:
```suggestion
- Pilah kontainer yang mengaitkan suatu volume tertentu atau memiliki volume yang terpasang pada jalur tertentu:
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Here are some reviews from mine.

**Review by reinhart1010 — APPROVED** [review]
Thanks! We will need to wait for another maintainer to approve and merge this PR.

**Comment by fadharpra** [comment]
Hi @kbdharun, kindly check again?

**Pull request merged**